modules/custom/drupalmel_core/drupalmel_core.install

```diff
@@ -81,6 +81,7 @@ function drupalmel_core_update_7000() {
   $feeds = array(
     'Kattekrab'   => 'http://kattekrab.net/taxonomy/term/322/feed',
     'Realityloop' => 'http://realityloop.com/tags/drupal-planet/feed',
+    'Flink' => 'http://flink.com.au/taxonomy/term/1/feed',
```

Comment:
Ah, I see how it works now. So:

    function drupalmel_core_update_7001() {
        $node       = new stdClass();
        $node->type = 'blog_feed';
        node_object_prepare($node);
        $node->title                               = 'Flink';
        $node->feeds['FeedsHTTPFetcher']['source'] = 'http://flink.com.au/taxonomy/term/1/feed';
        node_save($node);
    }